src/ufbdmp.f

```diff
@@ -1,6 +1,6 @@
 C> @file
 C> @author WOOLLEN @date 1994-01-06
-      
+C>      
```

Comment:
Ah ha! This will break things.

The reason is that there are actually two doxygen blocks. One, for the file, just has the author and date. The other, for the subroutine, has all the text.

If you put this 'C>' in, then doxygen will think that this is all file documentation, and there is no subroutine documentation.

So between the file documentation block and the first subroutine documentation block, there must be either a blank line, or a line with just 'C', but not a line with 'C>'.

This is subtle, and applies to many files.

Reply:
This I had no clue about, so I'll go ahead and fix that as well, and while I'm at it I'll also go back and correct this in the other 5 source files that I've already fully documented.